content/en/users/dev-env/binder/_index.md

```diff
@@ -5,9 +5,116 @@ weight: 20
 aliases:
   - /users/notebooks/kernels/binder
 description: >
-  Custom computing environments for EGI Notebooks
+  Custom reproducible computing environments for notebooks
 ---
 
 ## What is it?
 
-Repeatable, custom computing environments for notebooks
+Binder allows the re-creation of a custom computing environment for reproducible
+execution of notebooks (and potentially many other types of applications). Users
+who create their own notebooks in the [EGI Notebooks](../notebooks/)
+to analyze data can easily create a shareable link for those notebooks in the
+form of a GitHub repository. Based on this link, anyone can then reproduce the
+same data analysis using the link in the
+[EGI Binder](https://binder.notebooks.egi.eu) service.
+
+The service builds on [BinderHub](https://github.com/jupyterhub/binderhub), an
+Open Source tool that allows to build docker images from a Git repository and
+then make it available from JupyterHub.
```

Comment:
```suggestion
The service builds on [BinderHub](https://github.com/jupyterhub/binderhub), an
Open Source tool allowing to build docker images from a Git repository, and
then making them available via JupyterHub-based services like
[EGI Notebooks](../notebooks).
```

Reply:
Not sure about this, but I felt it would be good to be more concrete and link with our own service, if appropriate.

Reply:
I did a reword here, take a look